### ISSUE TITLE
508-defect-3 - design system, components  [KEYBOARD]: Focus outline MUST be visible around focused content, pagination

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8790]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
!  Team affected: VSA Public Websites; Project: Yellow Ribbon
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

For each page of search results, the focus outline must be visible around the entire page indicator. but is currently cropped at the top at bottom.

Previously documented: #8255, #8254

![Screen Shot 2020-04-06 at 10 52 15 AM (2)](https://user-images.githubusercontent.com/57469/78590291-9baefb00-780f-11ea-895a-bcc490225126.png)

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to tab through the page and identify where focus is.

**Comment by kelsonic** [comment]
This is an issue with the common `<Pagination />` component and is visible site-wide:

![image](https://user-images.githubusercontent.com/12773166/81441537-2ba5d480-912f-11ea-8fc5-99e6f714a606.png)

**Comment by jenstrickland** [comment]
This issue is not assigned to the Public Websites team, @kelsonic — it is recorded in the review where it was found, but assigned to VSP to address. Your team didn't need to do anything.

**Comment by kelsonic** [comment]
Oh! Well I sent up a PR for it anyways 🙂

**Comment by kelsonic** [comment]
Sorry @jenstrickland for self-assigning again! 😂I had already published `formation` with a fix for this, so I had self-assigned to do the last part which is bumping our `formation` version in `vets-website`, after which this issue should be resolved 🙂Hope that's ok but lmk if you'd prefer me to revert the fix for this

**Comment by jenstrickland** [comment]
The process agreed to with VSP is the issues that are `design system`, `components`, `cms`, and `forms` are VSP's responsibility to address. There are ramifications beyond the individual implementation that Rian Fowler wants to ensure are taken into account. 

If there are issues on your team's reviews that the title starts with `design system`, `components`, `cms`, and/or `forms` note those issues will not have the team's label assigned, and are not the team's responsibility. They should be left for VSP to address, so they can ensure all the consequences are evaluated.

**Comment by kelsonic** [comment]
Oh alright, I'll revert the fix then

**Comment by kelsonic** [comment]
@stephorkandatx Dropping this from current sprint as we can't work on it as the public websites team it sounds like

**Comment by kelsonic** [comment]
PRs closed:
https://github.com/department-of-veterans-affairs/vets-website/pull/12605
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/255

**Comment by ncksllvn** [comment]
Sorry @jenstrickland, we missed the distinction there on VSP vs. VSA owning this so we picked it up. It was a relatively easy one to knock out.

Just wondering - does VSP typically have bandwidth to grab issues that were spotted during a product's staging review but not tied directly to that product? I was thinking we could mark these issues as non-blocking for the product's launch, but the product team could possibly still own the work since it's still an enhancement to their product

**Comment by jenstrickland** [comment]
@ncksllvn - That is what I originally thought, as well. However, I was told that VSP needs to address the sitewide issues. I did ask that a process be defined so that VFS teams may contribute to fixes. VSP is spending the current few sprints working on evaluating all of the open issues, auditing the current tools, and either remediating or planning to address issues.

I have a large spreadsheet documenting all of the sitewide or vendor issues in the work of VSA teams. So, when I find a sitewide issue I note it in the spreadsheet and assign it to VSP. 

This all arose of out the issues discovered during other audits, when you said issues were sitewide and not your team's responsibility to fix. This pagination issue in the Yellow Ribbon work is the same as the pagination issue in Find Forms, too. I had a conversation with Rian and another with Trevor, and that's when the process was documented.

**Comment by ncksllvn** [comment]
Ah okay. Hm, I think I may have been suggesting to not block a product launch for an issue that is already live on the website (like a site-wide template issue). I didn't mean to defer our team's responsibility to fix an issue entirely though because I think it makes sense for us to fix them.

On Find Forms, there was an issue that entailed restructuring the site-wide HTML layout to fix the problem with the `<aside>`. I could see that specific example going to VSP because it is so impactful.

**Comment by jenstrickland** [comment]
It sounds like there are a lot of different items being discussed here. 

The "sitewide" issue with the `<aside>` element only exists in the CMS templates, not on the rest of VA.gov, and Kevin Walsh said the public websites team is responsible for / owns that template.

Another issue, adding the aria-labels to distinguish the main, breadcrumb, and secondary navigation items, was deemed a sitewide issue, so it was reassigned to VSP after my conversation with Rian.

This particular issue, #8790, is regarding the focus outline for sitewide pagination component. Any work to address this, as it was explained to me, needs to be QA'd thoroughly to understand what else it impacts (for example, how it renders in Android and other environments), and that is likely why VSP wants to handle it, since they intend to assess issues, fixes, and thoroughly QA.

**Comment by ncksllvn** [comment]
Hm so it can get pretty complicated to determine what team owns certain work. Either way, the work behind this issue is pretty low risk and we picked it up because it was attached to the Yellow Ribbon Epic. It's already done, so I don't want the effort to go to waste.

As an aside (😉), we make commits to the design system repo pretty regularly as part of our general workflow, so VSP may want to enforce jurisdiction and process via codeowners sometime to keep us from iterating there.

**Comment by kelsonic** [comment]
> we make commits to the design system repo pretty regularly as part of our general workflow, so VSP may want to enforce jurisdiction and process via codeowners sometime to keep us from iterating there

🎯💯That seems like the best solution to this since we work in the design system often as Public Websites

**Comment by jenstrickland** [comment]
Rian and I messaged in Slack, and he will talk with you Nick.

**Comment by ncksllvn** [comment]
Alrighty @jenstrickland , we'll be more careful next time to not pick up any issues that don't belong to us. We already completed this one though, so gonna mark it as such.

**Comment by jenstrickland** [comment]
I just validated that the focus outline is fixed, in all the browsers. Thanks for the great work, y'all! 🏆
